src/timed-stream.ts

```diff
@@ -79,7 +82,6 @@ export class TimedStream extends PassThrough {
         if (options?.transformHook) {
           options?.transformHook(event, _encoding, callback);
         }
-        callback(null, event);
```

Comment:
Calling the callback is now the responsibility of the transformHook. This is because in practice the transformHook uses the callback in different ways.